networkx/linalg/tests/test_laplacian.py

```diff
@@ -334,3 +334,126 @@ def test_directed_combinatorial_laplacian():
         nx.directed_combinatorial_laplacian_matrix(G, walk_type="pagerank", alpha=100)
     with pytest.raises(nx.NetworkXError):
         nx.directed_combinatorial_laplacian_matrix(G, walk_type="silly")
+
+
+class TestTotalSpanningTreeWeight:
```

Comment:
If these tests are identical to the ones added for `TestNumberOfSpanningTrees` then let's not add these here - the code is already tested there.